java_collection.go

```diff
@@ -17,10 +17,8 @@
 
 package hessian
 
-import (
-	perrors "github.com/pkg/errors"
-	"reflect"
-)
+import "reflect"
+import perrors "github.com/pkg/errors"
```

Comment:
change it in this way,

import (
    "reflect"
)

import (
     perrors "github.com/pkg/errors"
)